examples/tutorials/alpha-zero-tic-tac-toe/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I don't think you can drop in a LICENSE file like this without more explanation.

It is good you are including the LICENSE, but you might have to communicate in the README What files the LICENCE applies to, and what files fall under the Apache 2 License (of what applies to our organization)

Reply:
The only new python file is `DeterminedShim.py`, perhaps best if this LICENSE file is removed and the README updated tho.